GitUI/CommandsDialogs/FormCleanupRepository.cs

```diff
@@ -97,17 +110,32 @@ private CleanMode GetCleanMode()
             throw new NotSupportedException($"Unknown value for {nameof(CleanMode)}.");
         }
 
-        private string? GetPathArgumentFromGui()
+        private string? GetInclusivePathArgumentFromGui()
         {
-            if (!checkBoxPathFilter.Checked)
+            if (!checkBoxIncludePathFilter.Checked)
             {
                 return null;
             }
 
             // 1. get all lines from text box which are not empty
             // 2. wrap lines with ""
             // 3. join together with space as separator
-            return string.Join(" ", textBoxPaths.Lines.Where(a => !string.IsNullOrEmpty(a)).Select(a => string.Format("\"{0}\"", a)));
+            return string.Join(" ", textBoxIncludePaths.Lines.Where(p => !string.IsNullOrEmpty(p)).Select(p => string.Format("\"{0}\"", p)));
+        }
+
+        private string? GetExclusivePathArgumentFromGui()
+        {
+            if (!checkBoxExcludePathFilter.Checked)
+            {
+                return null;
+            }
+
+            // 1. get all lines from text box which are not empty
+            // 2. Prepend lines with '-e '
```

Comment:
```suggestion
            // 2. Prepend lines with '--exclude '
```

Same for tests -e there too.

I also would prefer to move this to GitCommandHelpers.cs, just providing the string  or an array (if checkbox set).
(This would affect tests too.)
But I leave this to another maintainer to request, I can live with this.